src/utils/typography.js

```diff
@@ -3,15 +3,27 @@ import CodePlugin from 'typography-plugin-code';
 import { MOBILE_MEDIA_QUERY } from 'typography-breakpoint-constants';
 
 const options = {
-  baseFontSize: '18px',
+  baseFontSize: '14px',
   baseLineHeight: 1.45,
   scaleRatio: 2.25,
+   googleFonts: [
+    {
+      name: 'Lato',
+      styles: [
+        '400',
+        '400i',
+        '900',
```

Comment:
Trailing commas here, we don't use these in this project.

Try running `npm run lint` before pushing to github 👍